src/components/heading/FilterStreets.jsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+
+import { Controller, useFormContext } from 'react-hook-form';
+
+import SelectStreet from './SelectStreet';
+
+const OPTIONS = [
+  { value: 'Belyye Rosy', name: 'Belyye Rosy' },
+  { value: 'Bolshaya Troitskaya', name: 'Bolshaya Troitskaya' },
+  { value: 'Dovatora', name: 'Dovatora' },
+  { value: 'Kabyaka', name: 'Kabyaka' },
+  { value: 'Lenina', name: 'Lenina' },
+  { value: 'Ozheshko', name: 'Ozheshko' },
+
+];
+
+function FilterStreets() {
```

Comment:
I think streets should be plain input with search value instead of select, we can expand this approach to autocomplete though

https://mui.com/material-ui/react-autocomplete/